examples/petsc/bps.c

```diff
@@ -45,6 +45,9 @@ const char help[] = "Solve CEED BPs using PETSc\n";
 #include <stdbool.h>
 #include <string.h>
 #include <petscksp.h>
+#ifdef PETSC_HAVE_CUDA
+#include <petsccuda.h>
+#endif
```

Comment:
With the merge of https://gitlab.com/petsc/petsc/merge_requests/2074, this header no longer exists. I would just protect this with `#if PETSC_VERSION_GE(3,12,0)` because it will soon be released (should work now with PETSc 'master').